x-pack/plugins/observability/public/pages/rule_details/index.tsx

```diff
@@ -43,30 +42,29 @@ import { usePluginContext } from '../../hooks/use_plugin_context';
 import { useFetchRule } from '../../hooks/use_fetch_rule';
 import { RULES_BREADCRUMB_TEXT } from '../rules/translations';
 import { PageTitle } from './components';
+import { useKibana } from '../../utils/kibana_react';
```

Comment:
Just curious, what is the reason of this change? What is the difference between `useKibana` from `'../../utils/kibana_react'` and `'@kbn/kibana-react-plugin/public'`?

Reply:
@maryam-saeidi, Thanks for bringing this up. Actually this part of the problem you mentioned here https://github.com/elastic/kibana/pull/135805#issuecomment-1194149972 

The `useKibana` should be pulled from https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/utils/kibana_react.ts, where we import all the necessary services for o11y in a centric way, and not construct that in each page.

But this [PR](https://github.com/elastic/kibana/pull/136601/files#diff-d19c39560cbd9661c7d9df71eed8fc45a188ada9bc97e0b7c321da8f4afdfdd6L45) imported the raw public `useKibana` from `@kbn/kibana-react-plugin/public` which is not our best option, and should have fixed that before approval.

Reply:
But I didn't notice the same issue in the other PR, so I am wondering why it caused an issue here 🤔

Reply:
Because it depends, the Rule Details page uses the new bulk actions in its Alerts table introduced in [PR](https://github.com/elastic/kibana/pull/136601/files#diff-d19c39560cbd9661c7d9df71eed8fc45a188ada9bc97e0b7c321da8f4afdfdd6L45). 
If another page want to use the Alert table with bulk actions, and import `useKibana` from https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/utils/kibana_react.ts they will get the same issue.

The takeaway point here, `userKibana` services/types should be defined in a central way in https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/utils/kibana_react.ts so we have consistency across the o11y pages.

If you like I can walk you through the issue over a Zoom call if it is still not very clear!